app/models/cloud_volume.rb

```diff
@@ -35,6 +37,15 @@ def self.class_by_ems(ext_management_system)
     ext_management_system && ext_management_system.class::CloudVolume
   end
 
+  def self.my_zone(ems)
+    # TODO(pblaho): find unified way how to do that
```

Comment:
where did this come from?

Reply:
https://github.com/ManageIQ/manageiq/blob/master/app/models/cloud_volume_snapshot.rb#L20-L27

Reply:
Not sure what he meant (or who he is) but didn't want to lose the TODO or change the behavior in CloudVolume vs CloudVolumeSnapshot